foods/models/CategoryModel.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+    parent_id = models.ForeignKey("self", related_name="categories", on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
need not to use Foreignkey, varchar is enoug, so, u can filter Cate by parent_id